games/ferium/Portfile

```diff
@@ -0,0 +1,334 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cargo 1.0
+PortGroup github    1.0
+
+github.setup        gorilla-devs ferium 4.7.0 v
+github.tarball_from archive
+
+maintainers         {johnlindop.com:git @JLindop} openmaintainer
+revision            0
+
+description         CLI program for downloading and updating Minecraft mods and modpacks.
+
+long_description    Ferium is a CLI tool for downloading and updating Minecraft mods from Modrinth \
+                    CurseForge, and GitHub Releases, and modpacks from Modrinth and CurseForge. \
+
+categories          games
+homepage            https://github.com/gorilla-devs/ferium
```

Comment:
`homepage` no longer needs to be explicitly set, since it is implicitly set by `github.setup`.